CRM/Csvimport/Import/Form/DataSource.php

```diff
@@ -59,10 +59,14 @@ public function buildQuickForm() {
     $allEntities = civicrm_api3('entity', 'get', array());
     $creatableEntities = array();
     foreach ($allEntities['values'] as $entity) {
-      $actions = civicrm_api3($entity, 'getactions', array('entity' => $entity));
-      //can add 'submit' later when we can figure out how to specify on submit
-      if(array_intersect(array('create',), $actions['values'])) {
-        $creatableEntities[$entity] = $entity;
+      try {
+        $actions = civicrm_api3($entity, 'getactions', array('entity' => $entity));
+        //can add 'submit' later when we can figure out how to specify on submit
+        if(array_intersect(array('create',), $actions['values'])) {
+          $creatableEntities[$entity] = $entity;
+        }
+      } catch (CiviCRM_API3_Exception $e) {
```

Comment:
Can you just fix the formatting to our coding standards ... we put the catch on the next line see https://www.drupal.org/coding-standards - also the random semi colon should go